### DISCUSSION TITLE
Setting up local_attributes.nc

### DISCUSSION THREAD

**Discussion by IanWhidden** [UW-Hydro/pysumma#170] [Q&A]
What relationship does the local_attributes.nc share with the forcing data? I see that the NetCDF file for local attributes "specifies model element attributes for GRUs and individual HRUs". I am running summa as a point-based simulation. How is the local_attribute.nc file used when creating NetCDF files for my forcing data? I have attached my code and output in an image below. 

<img width="972" alt="Screen Shot 2023-01-04 at 10 30 01 AM" src="https://user-images.githubusercontent.com/103448529/210617170-a4c059c6-e29d-4c34-820e-e9c999f16163.png">

**Comment by arbennett** [comment]
The local attributes are separate from the forcing data. It's technically possible to have the local attributes contained in the forcing netcdf data, but it's more common to have them as a separate netcdf file. You can see the required variables for the local attributes here: https://summa.readthedocs.io/en/latest/input_output/SUMMA_input/#attribute-and-parameter-files
